### ISSUE TITLE
steve jobs example MAKE FIRST TURN GO TO PAGE 3 OR 4?

### ISSUE THREAD

**Issue by csychester** [blasten/turn.js#385]
I am using the steve jobs version as a template.. how do i make the first turn go to another page other than 2? also is there a way to stop the cover page from reopening? e.g. after the book has been opened.. stop it from being closed?

Basically i am using side tabs and i dont want you to be able to navigate back to the front cover...

**Comment by balaurzor** [comment]
There is a VIEW option ( http://www.turnjs.com/docs/Property:_view ) that gives you back the current pages being viewed as an array,  if you want for it to stop the user from going back to the frontpage you will have to check which is your current view every time the next button is clicked and if it's [2,3] then stop it from turning using the stop() function. 

To make it open at another page just use the .turn('next'); function or .turn("page", pageNumber); in order to jump to another one.

Hope it helps

**Comment by csychester** [comment]
Cheers.. Thanks for your help..  Great work by the way!!!

**Comment by csychester** [comment]
sorry to ask but where do i place it in the current context. i want page 1 the front cover when opened to go to page 6...??
heres the current script from the steve-jobs example.. sorry i dont know javascript

if (currentPage>3 && currentPage<pages-3) {

```
                if (page==1 ) {
                    book.turn('page', 2).turn('stop').turn('page', page);
                    e.preventDefault();
                    return;
                } else if (page==pages) {
                    book.turn('page', pages-1).turn('stop').turn('page', page);
                    e.preventDefault();
                    return;
                }
            } else if (page>3 && page<pages-3) {
                if (currentPage==1) {
                    book.turn('page', 2).turn('stop').turn('page', page);
                    e.preventDefault();
                    return;
                } else if (currentPage==pages) {
                    book.turn('page', pages-1).turn('stop').turn('page', page);
                    e.preventDefault();
                    return;
                }
            }
```

What do i need to add or change??? thanks in advance

**Comment by balaurzor** [comment]
what exactly do you want to add or do here because I cannot understand . Do you just want to do the jump from the 1st page to the 6th or what ?

**Comment by csychester** [comment]
yes basically i want the cover page (1) to load as normal but the turn of page 1 to goto page 6 and ignore pages 2, 3, 4 and 5.

**Comment by balaurzor** [comment]
on the $('.next').click(function(){}) add this 
var view = $('.magazine').turn('view');
    if(view == '0,1'){
        $('.magazine').turn('page',6);
    }else{
               $('.magazine').turn('next'); 
        }

but this will handle the turn only when you click on the button , you will need to add the same thing when you peel the page
